### DISCUSSION TITLE
Video.exe download on load the web project

### DISCUSSION THREAD

**Discussion by MarkoShenodaa** [unoplatform/uno#16861] [Q&A]
After i build the solution when i try to start the web view it ask me to download file named "video.exe" and stop working i can't load the web page however all other platforms work fine @jeromelaban   @MartinZikmund 

![image](https://github.com/unoplatform/uno/assets/137176976/2685bdd3-1b21-4a65-97f5-72478663cd0a)

**Comment by agneszitte** [comment]
cc @jeromelaban, @MartinZikmund

**Comment by MartinZikmund** [comment]
@MarkoShenodaa can you please provide some repro? There is no video.exe included in Uno Platform, so it must be something specific with your setup

**Reply by jeromelaban** [reply]
@MarkoShenodaa If you can also [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html), as well as the debugger output from the browser, it will help.

**Reply by MarkoShenodaa** [reply]
https://github.com/MarkoShenodaa/UnoApp1

**Reply by jeromelaban** [reply]
I've launched the app from your sources, and I'm not reproducing the issue. Can you [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html), as well as the debugger output from the browser?

**Reply by MartinZikmund** [reply]
Could it be some browser extension causing this? @MarkoShenodaa which extensions are you using? Can you try a different browser?

**Reply by MarkoShenodaa** [reply]
i try it in a new installed browser with no extensions at all

**Reply by jeromelaban** [reply]
> i try it in a new installed browser with no extensions at all

Thanks, please provide logs, we're not able to reproduce.

**Reply by MarkoShenodaa** [reply]
how to get the logs

**Reply by MartinZikmund** [reply]
> I've launched the app from your sources, and I'm not reproducing the issue. Can you [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html), as well as the debugger output from the browser?

See the link

**Reply by MarkoShenodaa** [reply]
my it force install some browser extinctions one of them is the problem with uno web thanks i will provide the extension name

**Reply by MartinZikmund** [reply]
@MarkoShenodaa Please let us know, it is a very odd behavior indeed
